README.md

```diff
@@ -25,10 +25,20 @@ Note that some features involve bespoke interpretation of schema details suited
 For embedded usage the following will allow and support runtime schema conversion and table maintenance:
 
     const jsonSchemaBigquery = require('jsonschema-bigquery')
-    const bigquerySchema = jsonSchemaBigquery.run(jsonSchemaObject)
+    const bigquerySchema = jsonSchemaBigquery.run(jsonSchemaObject, options)
 
 Please ensure that the input JSON schema is dereferenced so that all external references have been resolved. [json-schema-ref-parser](https://www.npmjs.com/package/json-schema-ref-parser) can do this, prior to using this module.
 
+### Options
+
+```
+{
+  ignoreAdditional: true
+}
+```
+
+* `ignoreAdditional` - boolean, skip checking for additional properties in schemas.
```

Comment:
I don't think the naming (and description) of the option are explicit enough. I'm wondering if it wouldn't be better to go for a major version bump instead and leave the default be to not require `"additionalProperties": false` for `"type": "object"` properties and then rename the option to something like: `preventAdditionalObjectProperties`.

Reply:
major version bumped